simd/src/Kokkos_SIMD_Scalar.hpp

```diff
@@ -326,7 +326,7 @@ template <class T, class BinaryOp>
 [[nodiscard]] KOKKOS_FORCEINLINE_FUNCTION T
 reduce(const_where_expression<simd_mask<T, simd_abi::scalar>,
                               simd<T, simd_abi::scalar>> const& x,
-       T identity_element, BinaryOp) {
+       T identity_element, std::plus<>) {
```

Comment:
Did you mean to change?  `BinaryOp` template argument is not used any more and FTAD cannot kick in any more

Reply:
Ah good catch, thank you! I should've removed the template parameter as well. I'll do this now.